src/app/backend/client/api/common.go

```diff
@@ -16,19 +16,23 @@ package api
 
 import (
 	"crypto/rand"
+	"fmt"
+	"strings"
 
 	v1 "k8s.io/api/authorization/v1"
 )
 
 // ToSelfSubjectAccessReview creates kubernetes API object based on provided data.
-func ToSelfSubjectAccessReview(namespace, name, resource, verb string) *v1.SelfSubjectAccessReview {
+func ToSelfSubjectAccessReview(namespace, name, resourceKind, verb string) *v1.SelfSubjectAccessReview {
 	return &v1.SelfSubjectAccessReview{
 		Spec: v1.SelfSubjectAccessReviewSpec{
 			ResourceAttributes: &v1.ResourceAttributes{
 				Namespace: namespace,
 				Name:      name,
-				Resource:  resource,
-				Verb:      verb,
+				// Name must be in a plural lower-case format,
```

Comment:
Probably say "Resource name" instead of "Name" here. It's a bit confusing since "Name" could mean the line above.